frontend/jupyter/src/app/pages/form/form-default/form-image/form-image.component.html

```diff
@@ -122,6 +130,44 @@
     </mat-error>
   </mat-form-field>
 
+  <!-- If employeeOnly==false in config, or label in namespace==true, then display SAS image error message-->
+  <p 
+    style="color:red;"
+    *ngIf="(!parentForm?.value.customImageCheck && 
+    parentForm?.value.serverType === 'group-three' && 
+    nonEmployeeUsers())"
+  >
+    {{ 'jupyter.formImage.errorSAS' | translate }}
```

Comment:
Since the condition for enabled state of images is generic and configurable, so too should the messages be. To do this...

#### Add a field for admins
... to provide a message when the image is disabled/unavailable, supporting i18n. e.g.
```
enabledCondition:
  ...
disabledMessage:
  en: SAS is only for employees!!! >:(
  fr: Seulment pour les employes!!!
```

#### Use the active language to fetch the correct key
The translate module should let you do that; then you can display whatever was written in the config.

#### Fall back to default message

```
{{ 'jupyter.formImage.enabledConditionUnmet' | translate }}
```
Where the i18n strings JSON has something like:
```
"enabledConditionUnmet": "This workspace type is disabled for profile \"{{profile}}\"",
```